Libraries/Microsoft.Teams.Apps/AppBuilder.cs

```diff
@@ -95,6 +95,12 @@ public AppBuilder AddOAuth(string defaultConnectionName)
         return this;
     }
 
+    public AppBuilder AddContextExtra(IDictionary<string, object> extra)
```

Comment:
this is nice for global scenario, but I think they also want to be able to pass in custom data on each request, meaning we would need to also support this on a per request basis in the aspnetcore plugin and the `App.Process` method

Reply:
good catch - pushed commit